From: Daniele Tricoli <eriol@mornie.org>
Date: Wed, 4 Nov 2015 19:39:08 +0100
Subject: Don't rewrite git revision during build

Upstream uses git revision to report it inside version string. In Debian
we want to take it from source distribution to show same revision as upstream,
not Debian packaging repository revision.

Patch-Name: do_not_rewrite_git_revision_on_build.patch
---
 setup.py | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index 22a9c65..de7cdf9 100755
--- a/setup.py
+++ b/setup.py
@@ -51,9 +51,7 @@ def get_version_info():
     # write_version_py(), otherwise the import of pywt.version messes
     # up the build under Python 3.
     FULLVERSION = VERSION
-    if os.path.exists('.git'):
-        GIT_REVISION = git_version()
-    elif os.path.exists('pywt/version.py'):
+    if os.path.exists('pywt/version.py'):
         # must be a source distribution, use existing version file
         # load it as a separate module to not load pywt/__init__.py
         import types
